flink-runtime-web/src/main/resources/META-INF/NOTICE

```diff
@@ -6,35 +6,22 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- nvd3#1.8.4
+- rxjs
+- tinycolor2
+- tslib
 
 This project bundles the following dependencies under the MIT license. (https://opensource.org/licenses/MIT)
 See bundled license files for details.
 
-- angular:1.4.8
-- angular-drag-and-drop-list:1.4.0
-- angular-moment:0.10.3
-- angular-ui-router:0.2.15
-- bootstrap:3.3.6
-- dagre:0.7.5
-- dagre-d3:0.4.17
-- ev-emitter:1.1.1
-- font-awesome:4.5.0 (CSS)
-- graphlib:1.0.7
-- imagesloaded:4.1.4
-- jquery:2.2.0
-- lodash:3.10.1
-- moment:2.10.6
-- moment-duration-format:1.3.0
-- qtip2:2.2.1
-- Split.js:1.0.6
+- angular
+- ant-design-palettes
```

Comment:
Why did you remove the old dependencies from the list?
If we still ship the old UI, we will need to keep the dependency information from the old UI as well

Reply:
I forgot to move it back when adding the `old-verison` back, these should be kept.